ReactNativeTemplate/android/app/build.gradle

```diff
@@ -109,6 +109,11 @@ android {
             minifyEnabled enableProguardInReleaseBuilds
             proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
         }
+        debug {
+            defaultConfig {
+                targetSdkVersion 22
```

Comment:
We've had this for 2 releases now and no one has complained yet. I would prefer to leave it as is until we determine a proper fix. If someone complains or reaches out, we can ask them to drop the API version.